website/blog/2024-03-19-deploy-tabby-with-replicas-behind-reverse-proxy/index.md

```diff
@@ -0,0 +1,120 @@
+---
+authors: [meng]
+tags: [deployment, reverse proxy]
+---
+
+# Deploying Tabby with Replicas and a Reverse Proxy
+
+Welcome to our tutorial on how to set up Tabby, the self-hosted AI coding assistant, with Caddy serving as a reverse proxy (load balancer). This guide assumes that you have a Linux machine with Docker, CUDA drivers, and the [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html) already installed.
```

Comment:
Add a little bit of motivation here? What's the benefit of this? Why certain users would need it?

Reply:
added